Source/JavaScriptCore/wasm/WasmB3IRGenerator.cpp

```diff
@@ -359,9 +359,10 @@ class B3IRGenerator {
     PartialResult WARN_UNUSED_RETURN addI31New(ExpressionType value, ExpressionType& result);
     PartialResult WARN_UNUSED_RETURN addI31GetS(ExpressionType ref, ExpressionType& result);
     PartialResult WARN_UNUSED_RETURN addI31GetU(ExpressionType ref, ExpressionType& result);
+    ExpressionType WARN_UNUSED_RETURN pushArrayNew(uint32_t typeIndex, Value* initValue, ExpressionType size);
```

Comment:
Since this is purely a helper and doesn't need to be part of the interface of the generator, might make sense to put it in the private section below (where all the `emitFoo` declarations are).